lib/health_cards/jws.rb

```diff
@@ -24,7 +24,7 @@ def decode(data)
       # @param key [HealthCards::PrivateKey] the private key associated with the JWS
       # @return [HealthCards::HealthCard]
       def from_jws(jws, public_key: nil, key: nil)
-        header, payload, signature = jws.split('.').map { |entry| decode(entry) }
+        header, payload, signature = jws.to_s.split('.').map { |entry| decode(entry) }
```

Comment:
The documentation for this method says `jws` should be a string, so we shouldn't have to call `to_s` on it.

Reply:
This method should probably also accept a JWS object since we often have to chain calls to get the JWS and then the HealthCard objects, so recommend updating the documentation

Reply:
resolved